### DISCUSSION TITLE
loader animation

### DISCUSSION THREAD

**Discussion by Mr-Coquin** [moonshine-software/moonshine#1296] [Ideas]
Было бы прикольно иметь возможным задавать свой шаблон анимации ожидания. Например в beforRender пока он выполняется. Или же в кнопках или ресурсах, если он не в модалке открываются задать параметр своей анимации.

**Comment by lee-to** [comment]
Глобально менять лоадер или локально в каких то сущностях?

**Reply by Mr-Coquin** [reply]
Тут наверное не полностью сформулировал текст:
1) Хотелось бы быстро менять ка кто дизайн встроенного loader, а то в начале спрашивали что за Playstation или Игра в кальмара появилась. 
2)В некоторых местах прямо хочется при переходе на страницы, где загрузка большого пула данных идет вместо стандартного ожидания загрузки страницы повесить loader. Мы конечно обошли это сделав вызов Modal по нажатию на кнопку(ссылку) а там изменили вьюху Modal свою анимацию . Но это костыли :)

**Reply by lee-to** [reply]
1. Сделаю в 3 версии возможность заменить лоадер, пока это можно сделать просто переопределив вьюху, она не меняется поэтому можно смело менять
2. Тут уже вы самостоятельно делаете а костыль или нет тут уже зависит от вашего опыта в разработке
